test/misc/util.js

```diff
@@ -33,3 +35,51 @@ exports.file = function (hash) {
     file: hash
   }
 }
+
+exports.keysFor = function (name) {
+  const seed = crypto.createHash('sha256').update(name).digest()
+  return ssbKeys.generate(null, seed)
+}
+
+exports.readOnceFromDB = function (sbot) {
+  return new Promise((resolve) => {
+    var cancel = sbot.post((msg) => {
+      cancel()
+      resolve(msg)
+    }, false)
+  })
+}
+
+exports.trackProgress = function (sbot, name) {
+  let n = 0
+  let n0 = 0
+  sbot.post((msg) => n++)
+  setInterval(() => {
+    if (n0 !== n) {
+      exports.log(name, n, n - n0, sbot.progress())
+      n0 = n
+    }
+  }, 1000).unref()
+}
+
+exports.countClock = function (obj) {
+  let total = 0
+  let sum = 0
+  for (const key in obj) {
+    total++
+    sum += obj[key]
+  }
+  return { total, sum }
+}
+
+exports.randint = function (n) {
```

Comment:
can we call `n` `max` instead? I know the existing code had `n`, but I feel `max` reads better.